engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -43,14 +44,29 @@
     @Replicate
     public float lightAmbientIntensity = 1.0f;
 
+    /**
+     * This helps control how focused the specular light is. A smaller number will make a wider cone of light. A larger number will make a narrower cone of light.
+     */
     @Replicate
-    public float lightSpecularPower = 1.0f;
```

Comment:
I think both these images hint to the fact that more work needs to be done on the lighting model, as neither seems to be particularly realistic. I mean, wouldn't a torch a few meters from a wall light up a large section of the wall, similarly to what happens in the first image? On the other hand, wouldn't a torch near the floor light up a large portion of the floor too?